### ISSUE TITLE
Attack the right side of the tower

### ISSUE THREAD

**Issue by porglezomp** [FRC1076/FRC1076-2016-Competition#43]
As in not the left side

**Comment by a2aaron** [comment]
From Solomon's email:
- Program the robot to primarily attack the right side of the tower. We have given up our ability to go under the low bar in order to add the lift arm. Also, there will be a LOT of robots will do a low bar high goal routine. If we want to work well with them, we need to attack the opposite side of the tower. Plan on Defense positions 3-5, with an emphasis on positions 4 & 5. Rock Wall, Rough Terrain, Moat, Ramparts.

**Comment by a2aaron** [comment]
The constructor chain autonomous seems to attack the right side of the tower as well as the mini-compiler one. I don't know for sure though.

**Comment by porglezomp** [comment]
I'm pretty sure all the chains that I actually made useful attack the left side of the tower, not the right side. (We want right side from the driver's PoV, correct?)

**Comment by a2aaron** [comment]
You are correct about right side being on driver's PoV. Due to lack of time to rewrite and test everything (no practice field was at the competition and we didn't have time to check if the autonomous worked well), I made a temporary autonomous that simply drives backwards. I'll try to make it more advance if I can get time to test and refine it.

**Comment by a2aaron** [comment]
#57 technically fixes this by not shooting at all. (temp measure)
